packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -181,45 +232,66 @@ export const Reports = () => {
-                            )}
-                          </MenuItem>,
-                        ]}
-                      >
-                        <IconButton styleType="borderless">
-                          <SvgMore
-                            style={{
-                              width: "16px",
-                              height: "16px",
-                            }}
-                          />
-                        </IconButton>
-                      </DropdownMenu>
+                    selected={selectedReports.some((r) => report.id === r.id)}
+                    onClick={(e) => {
+                      if (e?.target?.className?.toString().startsWith("rcw-horizontal-tile-container")) {
+                        if (!e.ctrlKey)
+                          setSelectedReports([]);
+
+                        setSelectedReports((sr) =>
+                          sr.some((r) => report.id === r.id)
+                            ? sr.filter(
+                              (r) => report.id !== r.id
+                            )
+                            : [...sr, report]
+                        )
+                      }
+                    }
+                    }
+                    actionGroup={
+                      <div className="button-container">
+                        <ExtractionStatus
+                          state={extractionStates.get(report.id) ?? ExtractionStates.None}>
+                        </ExtractionStatus>
+                        <DropdownMenu
+                          menuItems={(close: () => void) => [
+                            <MenuItem
+                              key={0}
+                              onClick={() => {
+                                setSelectedReport(report);
+                                setReportsView(ReportsView.MODIFYING);
+                              }}
+                              icon={<SvgEdit />}
+                            >
+                              {ReportsConfigWidget.localization.getLocalizedString(
+                                "ReportsConfigWidget:Modify"
+                              )}
+                            </MenuItem>,
+                            <MenuItem
+                              key={1}
+                              onClick={() => {
+                                setSelectedReport(report);
+                                setShowDeleteModal(true);
+                                close();
+                              }}
+                              icon={<SvgDelete />}
+                            >
+                              {ReportsConfigWidget.localization.getLocalizedString(
+                                "ReportsConfigWidget:Remove"
+                              )}
+                            </MenuItem>,
+                          ]}
+                        >
+                          <IconButton styleType="borderless">
+                            <SvgMore
+                              style={{
```

Comment:
Does this style really override anything? In other PR this kind of style was removed for one icon.

Reply:
With this styleType the rendered button receives an additional css class `iui-borderless`. Therefore, the difference is visible.
![image](https://user-images.githubusercontent.com/74193579/191994756-38345934-0dc1-4b96-943e-567c1111f037.png)
![image](https://user-images.githubusercontent.com/74193579/191994698-10ae0494-410d-4bdd-92e4-b7839afe7d81.png)

Reply:
This icon should be passed through the `icon` prop in this component without the style override.

Reply:
My comment was for the `style` not `styleType`.

Reply:
style for SVG size has been removed